modernizer-maven-plugin/pom.xml

```diff
@@ -20,6 +20,10 @@
     <maven>3.0.5</maven>
   </prerequisites>
 
+  <properties>
+    <groovy.version>3.0.0-rc-3</groovy.version>
```

Comment:
Can we upgrade to the latest non-prerelease version?

Reply:
Groovy 2.5.x doesn't support the higher jdks.  Groovy 3 while not finalized is highly stable.  Its only major concern is that it requires java 8 to run with which is I belive the case here as well unless someone on the team is using less than jdk 8 to build.  Given this is only internally used to run the IT tests, its a safe upgrade.  I've been on 3.0.0-x for spotbugs maven plugin for quite some time in release versions.  If for no other reason, modernizer is raised to show it can build on travis with higher jdks, this will be necessary.

Reply:
groovy 3.x is now live.  Will be pushing 3.0.2 here shortly.  Thanks.